usr/share/rear/skel/default/etc/scripts/system-setup

```diff
@@ -116,9 +126,11 @@ if test -s "/md5sums.txt" ; then
             # In debug mode let the user confirm to proceed:
             read -p "Press ENTER to proceed 'bona fide' nevertheless "
         else
-            # In non-debug mode wait 10 seconds so that the user can read the md5sum output:
+            # In non-debug mode wait USER_INPUT_INTERRUPT_TIMEOUT (by default 30 seconds)
+            # so that the user can read and understand the md5sum output (could be several lines)
+            # unless in unattended_recovery mode where there is normally no user who reads something:
             echo -e "Proceeding 'bona fide' nevertheless...\n"
-            sleep 10
+            unattended_recovery || sleep $USER_INPUT_INTERRUPT_TIMEOUT
```

Comment:
I wonder whether is it wise in the unattended case to continue and don't even wait a bit. Since the unattended case is especially dangerous, shouldn't we be extra careful in case of inconsistencies?

Reply:
Currently we provide no means to the user
to actually do something when md5sums check failed
i.e. "Proceeding 'bona fide'" is currently hardcoded
so it does not help to wait a bit at that message
when noone is there to read that message.

In general I think in unattended mode there is no need
to wait anywhere for the user (because there is no user)
so in unattended mode all moves forward as fast as possible
until it finally succeeds or aborts with an error.

So in this particular case (md5sums check)
we could abort with an error in unattended mode
when the md5sums check did not fully succeed
but I think this would result a false abort
in most cases where "Proceeding 'bona fide'"
regardless of some failed md5sum check
would have succeeded to recreate the system
and if a failed md5sum check is crucial then
it would later abort with an error at the
place where the actual problem happens.